src/jit/lclvars.cpp

```diff
@@ -2847,6 +2847,15 @@ void Compiler::lvaSortByRefCount()
             lvaSetVarDoNotEnregister(lclNum DEBUGARG(DNER_PinningRef));
 #endif
         }
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+        else if (opts.MinOpts() && !JitConfig.JitMinOptsTrackGCrefs())
+        {
+            if (varTypeIsGC(varDsc->TypeGet()))
+            {
```

Comment:
Good point - in my testing version there's actually another statement in there so the two if's need to be separate but in the "real" version it's simple and might as well combine the conditions. Good catch, will include it in my next push.